### PR TITLE
[wasm] Disable some tests on NodeJS/Windows

### PR DESCRIPTION
**Pull request by radical**
Issue: https://github.com/dotnet/runtime/issues/73721

### CODE REVIEW

File: src/libraries/System.CodeDom/tests/System/CodeDom/Compiler/CodeCompilerTests.cs [hunk 1]
```diff
@@ -55,6 +55,7 @@ public void CompileAssemblyFromDom_NullOptions_ThrowsArgumentNullException()
 
         [Theory]
         [MemberData(nameof(CodeCompileUnit_TestData))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/73721", typeof(PlatformDetection), nameof(PlatformDetection.IsNodeJS), nameof(PlatformDetection.IsWindows))]
```
Comment (akoeplinger) [inline src/libraries/System.CodeDom/tests/System/CodeDom/Compiler/CodeCompilerTests.cs:58]: This won't work, the platform is Browser not Windows

Comment (maraf) [reply src/libraries/System.CodeDom/tests/System/CodeDom/Compiler/CodeCompilerTests.cs:58]: Yea, anyway I'll try to make a working version using https://nodejs.org/api/process.html#process_process_platform

**Comment by maraf** [comment]
I'll update the conditions.

File: src/libraries/System.CodeDom/tests/System/CodeDom/Compiler/CodeCompilerTests.cs (hunk 1 above)
Comment (radical) [reply src/libraries/System.CodeDom/tests/System/CodeDom/Compiler/CodeCompilerTests.cs:58]: Use platformdetection.IsBrowserOnWindows

**Review by pavelsavara — APPROVED** [review]

**Pull request merged**